packages/fxa-settings/src/pages/Signin/SigninTotpCode/index.tsx

```diff
@@ -174,7 +174,7 @@ export const SigninTotpCode = ({
       />
       <div className="mt-5 link-blue text-sm flex justify-between">
         <FtlMsg id="signin-totp-code-other-account-link">
-          <Link to={`/signin${location.search}`} className="text-start">
+          <Link to={`/${location.search}`} className="text-start">
```

Comment:
If we need to change this to `/`, it's going to need to be a hard navigate. Though, if someone's got something in local storage they're going to be redirected back to `/signin`.